assets/scss/utility/_display.scss

```diff
@@ -0,0 +1,19 @@
+$screenSizes: (
+  "md": "768"
+);
+
+@each $type in "none", "inline-block" {
+  .d-#{$type} {
+    display: #{$type};
+  }
+
+  @each $sizeKey, $sizeVal in $screenSizes {
+    .#{$sizeKey} {
+      &\:d-#{$type} {
+        @media screen and (min-width: #{$sizeVal}px) {
```

Comment:
I would invert this selection to have the media tag include all of its children selectors instead of the other way around; I'm unsure how CSS is processed and if it would end up registering multiple media listeners or not.